meilisearch-http/Cargo.toml

```diff
@@ -23,6 +23,7 @@ actix-web = "2"
 actix-web-macros = "0.1.0"
 chrono = { version = "0.4.11", features = ["serde"] }
 crossbeam-channel = "0.4.2"
+either = { version = "1.5.3", features = ["serde"] }
```

Comment:
I think you added this dependency when you thought about using `Either` in the `UserQuery::facet_filters` field, but you were not able to make that work and forget to remove it from there.